README.md

```diff
@@ -235,6 +235,7 @@ A few advantages of this solution:
 
 ## Develop
 
+1. Make sure you are using Node.js 14.0 or higher and npm 7.0 or higher
```

Comment:
```suggestion
1. Make sure you are using Node.js 14 or higher and npm 7 or higher
```

for devs it is normal that when you refer to any possible major version, you do not need to specify `.0` but just write `14`. Actually `14.0` is a bit confusing 😉

Reply:
gotcha, fixed it.